_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -222,16 +221,39 @@ class HealthCareApp extends React.Component {
       case 'financial-assessment':
         return {
           receivesVaPension: this.state.applicationData['personal-information']['va-information'].receivesVaPension,
-          neverMarried: this.state.applicationData['personal-information']['name-and-general-information'].maritalStatus === '' ||
-            this.state.applicationData['personal-information']['name-and-general-information'].maritalStatus === 'Never Married'
+          neverMarried: this.state.applicationData['personal-information']['name-and-general-information'].maritalStatus === '' || this.state.applicationData['personal-information']['name-and-general-information'].maritalStatus === 'Never Married'
```

Comment:
This has now been moved into `_health-care/_js/store/calculated.js`. Take a look at `SpouseInformatinoSection.jsx` to see how it can accessed.